PluginCode/src/main/java/smr/cs/ualberta/libcomp/action/ReplacementAction.java

```diff
@@ -51,25 +53,24 @@
  * The ActionReplacement class is the main action for the plugin
  * This is triggered by the replacement button on the main plugin dialog
  */
-
 public class ReplacementAction extends AnAction {
-    
-    public ArrayList<ImportStatement> ImportListObjects;
-    public ArrayList<DependencyStatement> DependListObjects;
-    public ArrayList<DependencyStatement> MavenListObjects;
+    public ArrayList<ImportStatement> importObjectList;
+    public ArrayList<DependencyStatement> dependObjectList;
```

Comment:
what is dependObjectList?

Reply:
gradle dependency object . what do you think `gradleObjectList`?

Reply:
updated